app/assets/javascripts/oneErrorMessageForPassword.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Created by elisahilprecht on 13/04/15.
+ */
+(function(){
+  var errorElement = document.getElementById('password1IDErrorText');
+  if(errorElement!=undefined){
+    errorElement.innerHTML = errorElement.innerHTML.split(',')[0];
+  }
+})();
```

Comment:
what does this code do? How do you know there are more than one error messages? Why don't you change the error messages on the Scala side instead of ignoring one through this function? That's not a very clean way to solve this issue.